akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ClusterShardingSettings.scala

```diff
@@ -219,6 +220,7 @@ final class ClusterShardingSettings(
     val snapshotPluginId: String,
     val stateStoreMode: String,
     val passivateIdleEntityAfter: FiniteDuration,
+    val shardRegionQueryTimeout: FiniteDuration,
```

Comment:
should probably be added to the `typed.ClusterShardingSettings` also, there you don't have to care about bin compat